source/includes/getting-started/_concepts.html.md

```diff
@@ -67,23 +73,30 @@ This means that you, the developer, can leverage your domain-specific expertise
 
 Inkling is designed to enable you to solve any AI problem whether learning to label images or controlling a simulation. In the current version, Bonsai's technology is focused on solving control and optimization problems using simulation. When you train a BRAIN to solve an AI problem, you write a file in Inkling that teaches the AI to solve it. Inkling files describe the concepts that you want to teach and the curriculum and lessons for teaching them.
 
- To learn more about Inkling, refer to our [Inkling Guide][2].
+To learn more about Inkling, refer to our [Inkling Guide][2].
+
+## What is the Bonsai CLI?
+
+The Bonsai Command Line Interface (CLI) is a command line tool that enables you to configure the Bonsai AI Engine. The CLI is especially useful for automation and connection to other tools. Currently, there are some actions that can only be performed using the CLI, such as loading your Inkling file and connecting your simulator.
 
-## What is the Bonsai Command Line Interface?
+To learn more about the Bonsai CLI, refer to our [CLI Reference][3].
 
-The Bonsai Command Line Interface (CLI) is a command line tool that enables you to configure the Bonsai Artificial Intelligence Engine. The CLI is especially useful for automation and connection to other tools. Currently, there are some actions that can only be performed using the CLI, such as loading your Inkling file and connecting your simulator.
+## What is beta.bons.ai for?
 
-## beta.bons.ai
+The [Bonsai Beta Website][4] is a browser-based tool for configuring and analyzing BRAINs stored in the Bonsai AI Engine. In the future, this website will be useful for sharing BRAINs, collaborating, and learning. There is some information you can only get from beta.bons.ai, such as a visualization of a BRAIN's training progress.
```

Comment:
Would we want to usually have [beta.bons.ai](https://beta.bons.ai) be a link? It seems weird to see a URL and have it not be a link to me but I could be convinced otherwise